src/Microsoft.PowerShell.GraphicalTools/OutGridviewCmdletCommand.cs

```diff
@@ -7,16 +7,16 @@
 using System.Collections.Generic;
 using System.Management.Automation;
 using System.Management.Automation.Internal;
-using OutGridView.Models;
+using GraphicalTools.Models;
 using System.Linq;
 
-namespace OutGridView.Cmdlet
+namespace GraphicalTools.Cmdlet
 {
     /// Enum for SelectionMode parameter.
     /// </summary>
     [Cmdlet(VerbsData.Out, "GridView", DefaultParameterSetName = "PassThru")]
     [Alias("ogv")]
-    public class OutGridViewCmdletCommand : PSCmdlet
+    public class GraphicalToolsCmdletCommand : PSCmdlet
```

Comment:
Don't want to change this. We still want this to be Out-GridView.

Reply:
Good catch, I'm still wrapping my head around that cmdlets are classes within the namespace and not methods/functions of the overall class.